src/main/java/org/folio/circulation/support/ValidationErrorFailure.java

```diff
@@ -80,7 +80,7 @@ private JsonObject asJson() {
 
   @Override
   public String toString() {
-    return String.format("Validation failure:%n%s",
+    return String.format("validation failure:%n%s",
```

Comment:
Purely readability. I made these changes so that the diagnostic messages produced during the tests (and hence, also the running system hopefully) read better